Source/Core/Core/IOS/FS/FileSystem.cpp

```diff
@@ -17,6 +18,13 @@ std::unique_ptr<FileSystem> MakeFileSystem(Location location)
   return std::make_unique<HostFileSystem>(nand_root);
 }
 
+IOS::HLE::ReturnCode ConvertResult(ResultCode code)
+{
+  if (code == ResultCode::Success)
+    return IPC_SUCCESS;
+  return static_cast<ReturnCode>(-(static_cast<s32>(code) + 100));
```

Comment:
It'd probably be a good idea to document this `+100` in some way (should've been in first place, probably. Chance is now, since you're moving the code around.)

Reply:
Done. I've added a comment to explain how this works